support/thirdparty/oc/pkg/cli/image/manifest/manifest.go

```diff
@@ -186,7 +186,7 @@ func ProcessManifestList(ctx context.Context, srcDigest digest.Digest, srcManife
 		}
 
 		switch {
-		case len(srcManifests) == 1 && !keepManifestList:
```

Comment:
Is always using the first one correct when len> 1 ?

Reply:
didn't find issue with that yet. At least it doesn't change current behavior and passed all my manual tests

Reply:
sounds great :)